rpcs3/rpcs3qt/main_window.cpp

```diff
@@ -38,10 +39,13 @@
 
 #include "rpcs3_version.h"
 #include "Emu/System.h"
+#include "Emu/IdManager.h"
 #include "Emu/system_config.h"
+#include "input_dialog.h"
```

Comment:
can you please move this to the other rpcs3qt includes above ?

Reply:
done.